clientconn.go

```diff
@@ -411,6 +433,9 @@ func (cc *ClientConn) Close() error {
 	}
 	cc.state = Shutdown
 	cc.stateCV.Broadcast()
+	if cc.events != nil {
+		cc.events.Finish()
```

Comment:
Sorry, I'm still getting used to this UI.  I've set cc.events = nil here and s.events = nil in server.go, after Finish.